framework/resources/Datasources/s3/fragments/fragments.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+source conf/drillTestConfig.properties
+
+mkdir ${DRILL_TEST_DATA_DIR}/Datasources/s3/minio/minio_data/tstbucket/tmp/big_files
+
+tar -xzmvf ${DRILL_TEST_DATA_DIR}/Datasources/ctas/decimal/fragments/decimal_big.tar.gz -C ${DRILL_TEST_DATA_DIR}/Datasources/s3/minio/minio_data/tstbucket/tmp/big_files
```

Comment:
Can you please share where these files are coming form? I had this question in the previous PR, but I don't think we got to it.

Reply:
These files are from another test suite **framework/resources/Functional/drill_fragments/decimal/**

Reply:
Since the data is checked in to GitHub, so i think there is no real dependency then..